subprojects/gradle/buildchecks/src/main/kotlin/com/avito/android/plugin/build_param_check/BuildChecksExtension.kt

```diff
@@ -57,11 +57,11 @@ open class BuildChecksExtension {
 
         open class AndroidSdk : Check(), RequireParameters {
 
+            @Deprecated("not used anymore, remove after 2020.19")
             var compileSdkVersion: Int = -1
```

Comment:
Let's remove it from the documentation: 
https://avito-tech.github.io/avito-android/docs/projects/buildchecks/#android-sdk-version